sagemaker-python-sdk/mxnet_gluon_sentiment/sentiment_analysis_with_mxnet_gluon.ipynb

```diff
@@ -209,7 +209,7 @@
    },
    "outputs": [],
    "source": [
-    "# m.delete_endpoint()"
+    "m.delete_endpoint()"
```

Comment:
I don't think we want to keep this method on the estimator longer term. Can you change this to use the session to delete it?

Reply:
Updated.